README.md

```diff
@@ -66,6 +66,52 @@ const componentsDefinition = getComponentsDefinitionJson();
 const componentSet = await parseDefinition(componentsDefinition);
 ```
 
+### generateComponentSetInfo
+
+To retrieve the component set information as provided when publishing an article from Studio, the `generateComponentSetInfo` function can be used.
+
```

Comment:
Suggestion:

The `generateComponentSetInfo` function can be used to generate a plain overview of components and component fields. This information is useful to interpret digital articles created for the given component set. The input for the function is information from the component set: component set info and rendition files (html, psv, etc).

The first parameter allows for specifying the components definition. As second parameter, a function is expected that returns the contents of a given component set file file as string.

Reply:
Updated the readme. Also with an example about the restrict children property.